### ISSUE TITLE
Ability to skip "checking support necessity" step of slicing when supports are disabled

### ISSUE THREAD

**Issue by ctrl-alt-breathe** [bambulab/BambuStudio#2445]
Having to wait for minutes on complicated prints for the step called "Checking support necessity" when supports are explicitly disabled is very annoying and adds a lot of time to the already sometimes long slicing process.
It would be very helpful to have something like a skip button during slicing or an option under support settings to disable this step.

**Comment by tangketan** [comment]
I'm also thinking of opening an option to disable it, but I'm afraid of too many options. How about re-using the option "slow down for overhangs"? If  "slow down for overhangs" is turned off, it means the users are certain the model has no overhangs, then there is no need to check support necessity.

**Comment by SaltWei** [comment]
> slow down for overhangs

No. They are different settings

**Comment by ctrl-alt-breathe** [comment]
Yeah, they are different. Even if it was something thrown in settings somewhere so it's not cluttering the print settings, it would be a huge help.

**Comment by kemper** [comment]
I would also love this feature. Some models this stage take abnormally long and I'm just playing around with infill and layer patterns and such.

**Comment by JansenSmith** [comment]
This basically adds hours to my workflow for no reason, given that I am explicitly disabling supports.

**Comment by vatogrande** [comment]
Seems like this is still not implemented?  I would love this feature
